src/shared/config/resultsDownload.ts

```diff
@@ -0,0 +1,42 @@
+import { FileFormat, ContentType } from '../types/resultsDownload';
+import { fileFormatsResultsDownload as fileFormatsResultsDownloadUniProtKB } from '../../uniprotkb/config/download';
+import { fileFormatsResultsDownload as fileFormatsResultsDownloadUniRef } from '../../uniref/config/download';
+import { Namespace } from '../types/namespaces';
+
+export const fileFormatToContentType: Record<FileFormat, ContentType> = {
+  [FileFormat.fastaCanonical]: ContentType.fasta,
+  [FileFormat.fastaCanonicalIsoform]: ContentType.fasta,
+  [FileFormat.fastaRepresentative]: ContentType.fasta,
+  [FileFormat.tsv]: ContentType.tsv,
+  [FileFormat.excel]: ContentType.excel,
+  [FileFormat.xml]: ContentType.xml,
+  [FileFormat.rdfXml]: ContentType.rdfXml,
+  [FileFormat.text]: ContentType.text,
+  [FileFormat.gff]: ContentType.gff,
+  [FileFormat.list]: ContentType.list,
+  [FileFormat.json]: ContentType.json,
+};
+
+export const fileFormatToUrlParameter: Record<FileFormat, string> = {
+  [FileFormat.fastaCanonical]: 'fasta',
+  [FileFormat.fastaCanonicalIsoform]: 'fasta',
+  [FileFormat.fastaRepresentative]: 'fasta',
+  [FileFormat.tsv]: 'tsv',
+  [FileFormat.excel]: 'xlsx',
+  [FileFormat.xml]: 'xml',
+  [FileFormat.rdfXml]: 'rdf',
+  [FileFormat.text]: 'txt',
+  [FileFormat.gff]: 'gff',
+  [FileFormat.list]: 'list',
+  [FileFormat.json]: 'json',
+};
+
+export const fileFormatsWithColumns = [FileFormat.tsv, FileFormat.excel];
+
+export const nsToFileFormatsResultsDownload: Partial<Record<
```

Comment:
I guess the Partial here is supposed to be temporary, otherwise TypeScript would complain about exhaustivity of the Record.
We'll need to make sure to update them when we do add more namespace.
In other places we have put all the namespace, but the value either null, or empty string when not implemented yet. But it's also not a solution to have us remember about it later.
I'd say add a `// TODO:` comment before this and other constructs like these for now, not sure if there's a better way for us to remember to change the code and remove `Partial` eventually.

Reply:
Good idea - added some comments for our future selves.